test/functional/fixtures/api/es-next/selector/test.js

```diff
@@ -271,6 +275,51 @@ describe('[API] Selector', function () {
                     });
             }
         );
+
+        it('Should raise error if addCustomMethods method argument is not object',
```

Comment:
There is no need to abuse functional tests for these type checks. Use unit tests instead